Lib/test/test_smtplib.py

```diff
@@ -1289,6 +1291,7 @@ def test_main(verbose=None):
         SMTPAUTHInitialResponseSimTests,
         SMTPSimTests,
         TooLongLineTests,
+        SMTPUTF8SimTests,
```

Comment:
Rather than add to this list, let's modernize this test file.  It doesn't do anything special in test_main, so we can just delete test_main and make the `if __name__ == '__main__'` clause call `unittest.main()`, and regrtest/unittest will find all the tests.